README.md

```diff
@@ -50,69 +52,39 @@ The ingress host names are as follows, where `<base_hostname>` is your deploymen
 * Optional:
   Installation of the `podman` and [`cosign`](https://github.com/sigstore/cosign) binaries to verify that the RHTAS service is working as expected.
 
-## Deploying
+## Installation
 
-1. Create an `inventory` file with a single node under the `rhtas` group:
-   
-   ```
-   [rhtas]
-   123.123.123.123
-   ```
 
-2. Create an Ansible Playbook named `play.yml`, and replace `TODO` with your relevant information:
-   
-   ```yaml
-   - hosts: rhtas
-     vars:
-       base_hostname: TODO # e.g. example.com
-       # access credentials for registry.redhat.io (https://access.redhat.com/RegistryAuthentication)
-       tas_single_node_registry_username: TODO
-       tas_single_node_registry_password: TODO
-       tas_single_node_oidc_issuers:
-         - issuer: TODO # your OIDC provider (e.g. keycloak) URL
-           client_id: trusted-artifact-signer
-           url: TODO # your OIDC provider (e.g. keycloak) URL
-           type: email
-     tasks:
-       - name: Include TAS single node role
-         ansible.builtin.include_role:
-           name: redhat.artifact_signer.tas_single_node # Use if deploying from Ansible Automation Hub.
-         vars:
-           ansible_become: true
-   ```
-   > [!NOTE]
-   If running this Playbook from a locally-cloned Git repository, then replace the `redhat.artifact_signer.tas_single_node` value with `tas_single_node`.
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
 
-3. Install the RHTAS Ansible collection.
-   
-   - If installing from Ansible Automation Hub, then run the following command:
-   
-     ```shell
-     ansible-playbook -i inventory play.yml
-     ```
+```
+ansible-galaxy collection install redhat.artifact_signer
+```
 
-   - If running from a locally-cloned Git repository, then run the following command:
-   
-     ```shell
-     export ANSIBLE_ROLES_PATH="roles/" ; ansible-playbook -i inventory play.yml
-     ```
+You can also include it in a requirements.yml file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+
+```yaml
+collections:
+  - name: redhat.artifact_signer
+```
+
+Note that if you install any collections from Ansible Galaxy, they will not be upgraded automatically when you upgrade the Ansible package.
```

Comment:
We don't want to use passive voice in technical documentation.  Please rewrite this sentence in active voice.  For example:

Installing any collections from Ansible Galaxy does not upgrade the Ansible packages automatically.

Reply:
Hi Aron, unfortunately this is a part of the boilerplate that the Ansible team wants to have as-is in all readmes. We can bring this suggestion with them, but I think for now we have to keep this.